.travis.yml

```diff
@@ -1,17 +1,22 @@
 sudo: required
-
 language: java
-
 services:
-  - docker
-
+- docker
 script:
-  - mvn clean install -Ddocker.showLogs -Pdocker -Pjacoco -Pit -Pcoverage coveralls:report
-
+- mvn clean install -Ddocker.showLogs -Pdocker -Pjacoco -Pit -Pcoverage coveralls:report
 after_success:
-  - mvn deploy -DskipTests --settings .travis.settings.xml
```

Comment:
we need snapshots deployed too on each build, in order to allow testing in demo projects to discover potential bugs before release

Reply:
Can we just keep deploying the snapshot there?

Reply:
Currently I have made the changes to retain old behaviour and on tagged commit the auto signing will be performed for deploy.